builder/default.conf

```diff
@@ -0,0 +1,23 @@
+server {
```

Comment:
Is this the file you intend to overwrite, do you want to create another one next to it, or do you want to be able to overwrite the `nginx.conf` file below?

Reply:
I don't think it matter that much but I intended to inject a separate file for internal use that configures SSL.